azkaban-exec-server/src/main/java/azkaban/execapp/JettyServerModule.java

```diff
@@ -0,0 +1,77 @@
+package azkaban.execapp;
+
+import static azkaban.execapp.GuiceConstants.EXEC_JETTY_SERVER;
+import static azkaban.execapp.GuiceConstants.EXEC_ROOT_CONTEXT;
+
+import azkaban.utils.Props;
+import com.google.inject.AbstractModule;
+import com.google.inject.Provides;
+import javax.inject.Named;
+import javax.inject.Singleton;
+import org.apache.log4j.Logger;
+import org.mortbay.jetty.Connector;
+import org.mortbay.jetty.Server;
+import org.mortbay.jetty.servlet.Context;
+import org.mortbay.jetty.servlet.ServletHolder;
+import org.mortbay.thread.QueuedThreadPool;
+
+public class JettyServerModule extends AbstractModule {
```

Comment:
So package name `execapp` is not enough? I can rename it of course if you say so.